packages/xy-chart/src/utils/drag/Drag.js

```diff
@@ -0,0 +1,115 @@
+import React from 'react';
```

Comment:
I think you'll get some linting errors for `Drag.js` not having a `.jsx` name. Also similar to brush we could move this to a `src/components` folder.